spotify_os.xcconfig

```diff
@@ -62,6 +62,15 @@ HF_IGNORED_WARNINGS_0820 = $(HF_IGNORED_WARNINGS_0810)
 
 WARNING_CFLAGS = -Weverything -Wno-error=deprecated -Wno-objc-missing-property-synthesis -Wno-gnu-conditional-omitted-operand -Wno-gnu -Wno-reserved-id-macro -Wno-auto-import -Wno-missing-variable-declarations -Wno-vla -Werror $(HF_IGNORED_WARNINGS_$(XCODE_VERSION_MINOR))
 
+// Apple LLVM 8.0 - Warnings - All languages
```

Comment:
@JohnSundell You have a better memory than I do. We’re not actually using ios-ci for Hubs (more like ios-ci v2.0). So will have to update in both places.

@8W9aG I’ve sent a PR for ios-ci: https://github.com/spotify/ios-ci/pull/19